Denisov Alexey Streamlit App

denisovalex opened the pull request.

denisovalex committed:
main.py and requirements.txt added

SpirinEgor reviewed (changes requested):
You need to refactor this a little bit:
1. Avoid using global `PIPE`. Since you are caching it, you can easily call the function at any time.
2. Remove code from `if __name__ == "__main__"` under a separate function (it is not necessary, but highly recommended) 

Also, great functionality is to generate multiple images from one prompt, they always different. And you can add a download button for quick saving images.

denisovalex committed:
Structure changed

Global PIPE removed, code from <if __name__ == "__main__"> removed to <start_streamlit> function

denisovalex committed:
Minor bug fixed

denisovalex committed:
New user interface added

Now user can generate multiple images and save them

denisovalex committed:
fp16 returned

fp 16 had been removed for testing and returned now

SpirinEgor reviewed (approved):
👍
